framework/src/utils/PerfGraph.C

```diff
@@ -21,10 +21,13 @@
 // System Includes
 #include <chrono>
 
-PerfGraph::PerfGraph() : _current_position(0), _active(true)
+const std::string PerfGraph::ROOT_NAME = "Root";
+
+PerfGraph::PerfGraph(const std::string & root_name)
+  : _root_name(root_name), _current_position(0), _active(true)
 {
   // Not done in the initialization list on purpose because this object needs to be complete first
-  _root_node = libmesh_make_unique<PerfNode>(registerSection("App", 0));
+  _root_node = libmesh_make_unique<PerfNode>(registerSection(ROOT_NAME, 0));
```

Comment:
I really don't see a reason to change this.  Why is "Root" better than "App"?  "App" is what is printed in the graph... (and is already in use in some of my personal scripts - I'm sure others are using it too).